internal/types/path.go

```diff
@@ -67,7 +76,7 @@ func (p Path) Slice() []string {
 
 // Suffix returns the last path element.
 func (p Path) Suffix() string {
-	if len(p.s) <= 1 {
+	if len(p.s) < 1 {
```

Comment:
That makes it different from Prefix, TrimSuffix, TrimPrefix below. And I think filterDocumentPair depends in the current behavior. Can we avoid that change?

And let's add unit tests so we don't break that behavior

Reply:
Tests added, condition restored.